data-prepper-plugins/http-sink/src/integrationTest/java/org/opensearch/dataprepper/plugins/sink/HttpSinkServiceIT.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.sink;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.dataformat.yaml.YAMLGenerator;
+import io.micrometer.core.instrument.Counter;
+import org.apache.hc.client5.http.impl.classic.HttpClientBuilder;
+import org.apache.hc.client5.http.impl.classic.HttpClients;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.opensearch.dataprepper.metrics.PluginMetrics;
+import org.opensearch.dataprepper.model.codec.OutputCodec;
+import org.opensearch.dataprepper.model.configuration.PipelineDescription;
+import org.opensearch.dataprepper.model.configuration.PluginModel;
+import org.opensearch.dataprepper.model.configuration.PluginSetting;
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.event.EventHandle;
+import org.opensearch.dataprepper.model.event.JacksonEvent;
+import org.opensearch.dataprepper.model.log.JacksonLog;
+import org.opensearch.dataprepper.model.plugin.PluginFactory;
+import org.opensearch.dataprepper.model.record.Record;
+import org.opensearch.dataprepper.plugins.accumulator.BufferFactory;
+import org.opensearch.dataprepper.plugins.accumulator.InMemoryBufferFactory;
+import org.opensearch.dataprepper.plugins.sink.configuration.HttpSinkConfiguration;
+import org.opensearch.dataprepper.plugins.sink.configuration.ThresholdOptions;
+import org.opensearch.dataprepper.plugins.sink.dlq.DlqPushHandler;
+import org.opensearch.dataprepper.plugins.sink.service.HttpSinkService;
+import org.opensearch.dataprepper.test.helper.ReflectivelySetField;
+
+import java.text.MessageFormat;
+import java.time.Duration;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.UUID;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.opensearch.dataprepper.plugins.sink.service.HttpSinkService.HTTP_SINK_RECORDS_SUCCESS_COUNTER;
```

Comment:
Same here.
```suggestion
import static org.opensearch.dataprepper.plugins.sink.http.service.HttpSinkService.HTTP_SINK_RECORDS_SUCCESS_COUNTER;
```

Reply:
Thanks for the review @oeyh , Sure I shall fix it